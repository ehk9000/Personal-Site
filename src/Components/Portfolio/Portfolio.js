import React from 'react';

const Portfolio = () => {
    return (
      <section>
        <h2>My Portfolio</h2>
        <article>
          <img src={require("../../assets/images/CoderDojo.png")} alt="Splash Page for CoderDojo Website"/>
          <img src={require("../../assets/images/TrapperKeeper.png")} alt="Splash Page for TrapperKeeper"/>


        </article>
  
      </section>
    )
}

export default Portfolio;