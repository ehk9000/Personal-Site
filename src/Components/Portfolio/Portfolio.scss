.portfolio-container {
  display: flex;
  flex-flow: row nowrap;
}

img {
  height: 50px;
  width: 50px;
}