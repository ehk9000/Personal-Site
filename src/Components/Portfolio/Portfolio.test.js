import React from 'react';
import { shallow } from 'enzyme';
import Portfolio from './Portfolio';

describe('Portfolio', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Portfolio />)
  });

  it('Should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

