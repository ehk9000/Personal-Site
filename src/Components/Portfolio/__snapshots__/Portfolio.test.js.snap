// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio Should match snapshot 1`] = `ShallowWrapper {}`;
